python/examples/example_steadystate/model_steadystate_scaled.xml

```diff
@@ -35,15 +35,15 @@
       <parameter id="p4" value="2" units="per_second" constant="true"/>
       <parameter id="p5" value="0.1" units="per_second" constant="true"/>
       <parameter id="k4" value="1" units="mole_per_second" constant="true"/>
-      <parameter id="observable_x1" value="1" constant="true"/>
-      <parameter id="observable_x2" value="1" constant="true"/>
-      <parameter id="observable_x3" value="1" constant="true"/>
+      <parameter id="observable_x1" value="1" constant="false"/>
+      <parameter id="observable_x2" value="1" constant="false"/>
+      <parameter id="observable_x3" value="1" constant="false"/>
       <parameter id="scaling_x1" value="1" constant="true"/>
-      <parameter id="observable_x1_scaled" value="1" constant="true"/>
+      <parameter id="observable_x1_scaled" value="1" constant="false"/>
       <parameter id="offset_x2" value="1" constant="true"/>
-      <parameter id="observable_x2_offsetted" value="1" constant="true"/>
-      <parameter id="observable_x1withsigma_sigma" value="1" constant="true"/>
-      <parameter id="observable_x1withsigma" value="1" constant="true"/>
+      <parameter id="observable_x2_offsetted" value="1" constant="false"/>
+      <parameter id="observable_x1withsigma_sigma" value="1" constant="false"/>
```

Comment:
This one is the sigma parameter and could remain constant. I don't see a problem having it non-constant, though.

Reply:
true!